.openshift-ci/get_latest_helm_chart_versions.py

```diff
@@ -84,6 +96,69 @@ def get_latest_helm_chart_version_for_specific_release(chart_name, release):
         remove_helm_repo()
 
 
+def get_supported_helm_chart_versions():
+    add_helm_repo()
+    try:
+        update_helm_repo()
+        return __get_supported_helm_chart_versions()
+    finally:
+        remove_helm_repo()
+
+
+def __get_supported_helm_chart_versions():
+    supported_central_versions = []
+    supported_sensor_versions = []
+
+    supported_releases = get_supported_releases()
+    for release in supported_releases:
+        if __does_chart_exist("stackrox-central-services", release):
+            supported_central_versions.append(__get_latest_helm_chart_version_for_specific_release(
+                "stackrox-central-services", release)
+            )
+        else:
+            logging.debug(f"Supported version \"{release.major}.{release.minor}\" has no corresponding helm chart for "
+                          f"stackrox-central-services.")
+        if __does_chart_exist("stackrox-secured-cluster-services", release):
+            supported_sensor_versions.append(__get_latest_helm_chart_version_for_specific_release(
+                "stackrox-secured-cluster-services", release)
+            )
+        else:
+            logging.debug(f"Supported version \"{release.major}.{release.minor}\" has no corresponding helm chart for "
+                          f"stackrox-secured-cluster-services.")
+    return supported_central_versions, supported_sensor_versions
+
+
+def get_supported_releases():
+    supported_releases = []
+    data = __get_data_from_api(PRODUCT_LIFECYCLES_API)
+    releases = data["data"][0]["versions"]
+    for release in releases:
+        if release["type"] != "End of life":
+            supported_releases.append(parse_release(release["name"]))
+    return supported_releases
+
+
+def __get_data_from_api(url):
+    req = Request(
+        url=url,
+        headers={'User-Agent': 'Mozilla/5.0'}
+    )
+    with urlopen(req) as response:
+        response_bytes = response.read()
+        response_string = response_bytes.decode('utf-8')
+        data = json.loads(response_string)
+        return data
```

Comment:
I would also make sure this success before returning.

Reply:
Sure, I'll add some exception handling.